command.go

```diff
@@ -0,0 +1,120 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sdk
+
+import (
+	"context"
+
+	"github.com/conduitio/conduit-commons/opencdc"
+)
+
+type Command interface {
```

Comment:
I thought the command is only relevant when the connector is running as a standalone WASM processor, I think it would make sense to move this into `wasm`. If the reason for this was to make it accessible to Conduit, see my other comments re exposing `wasm` 🙃

Reply:
Yeah, I've intentionally started with as much as possible in `internal`, so we reduce the public API. But agreed, exposing `wasm` makes sense.

Reply:
I exposed `wasm` now and simplified how we use commands, it's all going directly through the proto structs so there's less conversions, less boilerplate code. Also made it so that the byte buffer is reused when communicating with Conduit.